test/plugins/RoutingPlugin.test.js

```diff
@@ -48,7 +51,8 @@ describe('RoutingPlugin', () => {
 		$injector
 			.registerSingleton('RoutingService', routingService)
 			.registerSingleton('TranslationService', translationService)
-			.registerSingleton('EnvironmentService', environmentService);
+			.registerSingleton('EnvironmentService', environmentService)
+			.registerSingleton('UnitsService', unitsServiceMock);
```

Comment:
Why do we have to change the `RoutingPlugin` test setup when just the `RouteChart` was modified?